healthbuddy_backend/rapidpro/views.py

```diff
@@ -175,3 +182,20 @@ def filter_queryset(self, queryset):
         end_date = query_params.get("end_date", "2100-01-01")
         queryset = queryset.filter(day__range=[start_date, end_date])
         return super().filter_queryset(queryset).order_by("day")
+
+
+class DailyLabelCountListView(ListAPIView):
+    queryset = DailyLabelCount.objects.all()
+    model = DailyLabelCount
+    pagination_class = None
+    serializer_class = DailyLabelCountSerializer
+    filterset_fields = ["label__uuid", "label__name", "day"]
+    search_fields = ["label__uuid", "label__name", "day"]
+    ordering_fields = ["label__uuid", "label__name", "day"]
+
+    def filter_queryset(self, queryset):
```

Comment:
In the future we will be able to create a mixin that is responsible for implementing the filter by date range, and in the CBVs we would only need to inherit from that mixin.

Reply:
It's a crucial suggestion. I will probably work on it the next time I work on this project.